graphistry/Plottable.py

```diff
@@ -68,9 +83,6 @@ class Plottable(object):
     _bolt_driver : Any
     _tigergraph : Any
     
-    _spanner_config: Optional[SpannerConfig]
-    _kusto_config: Optional[KustoConfig]
```

Comment:
why did these go away?

Reply:
I broke types and didn't want to figure out, my todo includes adding these back